aws/sdk/integration-tests/s3/tests/concurrency.rs

```diff
@@ -22,17 +22,21 @@ use tokio::sync::Semaphore;
 use tokio::time::{Duration, Instant};
 use tracing::debug;
 
-const TASK_COUNT: usize = 10_000;
+// WARNING:
+// When testing this on your own computer, be sure to run the tests in several different terminals.
+// Depending on the terminal used, you may run into errors related to "Too many open files".
+
+const TASK_COUNT: usize = 1_000;
```

Comment:
Worth suggesting to set `ulimit` to a higher threshold?

Reply:
Eh, I'd rather ensure that the defaults don't require that at all

Reply:
That's definitely the ideal scenario, but you never know what configuration people might have